src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.stories.tsx

```diff
@@ -0,0 +1,91 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { DataList } from '../../../../../core';
+import { MemberDataListItemStructure } from './memberDataListItemStructure';
+
+const meta: Meta<typeof MemberDataListItemStructure> = {
+    title: 'Modules/Components/Member/MemberDataListItem.Structure',
+    component: MemberDataListItemStructure,
+    tags: ['autodocs'],
+    parameters: {
+        design: {
+            type: 'figma',
+            url: 'https://www.figma.com/file/ISSDryshtEpB7SUSdNqAcw/branch/P0GeJKqILL7UXvaqu5Jj7V/Aragon-ODS?type=design&node-id=14385%3A30819&mode=dev',
+        },
+    },
+    argTypes: {
```

Comment:
The component properties should already be recognised by Storybook, I wouldn't specify all the properties for all the components on the stories